src/Console/Commands/BassetClear.php

```diff
@@ -42,6 +42,7 @@ public function handle(): void
 
         $disk->deleteDirectory($path);
         $disk->makeDirectory($path);
+        $disk->delete('.basset');
```

Comment:
So this is something I wanted to talk to you about... Should we use the _singular_ name everywhere in the file paths... OR the _plural_? 👀 🤷‍♂️ Would it make more sense to be:
- `storage/app/public/basset` (or)
- `storage/app/public/bassets`

I mean... are we storing 
- "_things that Basset uses_" (or)
- "_backed up assets (aka bassets)_"

Which one sounds more intuitive to you?

Reply:
Good questions!
In the beggining we were using `bassets`, at some point (probably while creating the package) you named it `basset`, and from that point I started to call Basset to every of this "global" thigs.

I though that `.basset` is the file that stores Basset related stuff 🤷‍♂️
One day it may store more than just the map of bassets ...

`basset` is the folder named after the package name, that stores anything related with basset package ...

This is what I thought, `basset` works for everything, `bassets` only to mention assets "backed up assets" ...

Let me know if we should shift 😅

Reply:
Ok fine, I guess 😅 We can do a full code review after we know we're done. Maaaybe 😀